### DISCUSSION TITLE
Unable to Install Big Sur

### DISCUSSION THREAD

**Discussion by bocah-ndeso** [macOS-Dell-Latitude-E7470/macOS-Dell-Latitude-E7470#34] [Q&A]
I copied the EFI provided here. It booted perfectly. But always unable to install Big Sur. Once boot Mac Installer (big sur) is selected, it ended up to recovery screen (the same with first screen to download install file from Apple). Is there special steps needed or another prerequisite configuration needed?

Your advice is greatly appreciated.

**Comment by adityabakare** [comment]
Make sure your boot mode is set to AHCI and not RAID

**Reply by bocah-ndeso** [reply]
Yes, the SATA setting is set as ACPI. And boot security disabled.
